lte/gateway/c/sctpd/sctpd_event_handler.h

```diff
@@ -20,24 +20,23 @@
 namespace magma {
 namespace sctpd {
 
-// Sctp handler that relays events to MME over GRPC
+// Sctp handler that relays events to MME/AMF over GRPC
 class SctpdEventHandler : public SctpEventHandler {
  public:
-  // Construct SctpdEventHandler that communicates to MME over client
-  explicit SctpdEventHandler(SctpdUplinkClient& client);
+  // Construct SctpdEventHandler that communicates to MME/AMF over client
+  explicit SctpdEventHandler(SctpdUplinkClient &client);
```

Comment:
Revert this change as the style convention in Magma is to align & with the data type
```suggestion
  explicit SctpdEventHandler(SctpdUplinkClient& client);
```